apis/management.cattle.io/v3/multi_cluster_app_types.go

```diff
@@ -13,3 +16,67 @@ type GlobalDNS struct {
 	TTLSeconds   int64  `json:"ttl" norman:"default=300"`
 	ProviderName string `json:"providerName,omitempty" norman:"required"`
 }
+
+type MultiClusterApp struct {
+	types.Namespaced
+
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   MultiClusterAppSpec   `json:"spec,omitempty"`
+	Status MultiClusterAppStatus `json:"status,omitempty"`
+}
+
+type MultiClusterAppSpec struct {
+	ReleaseName string             `json:"releaseName,omitempty"`
+	Answers     map[string]string  `json:"answers,omitempty"`
+	HelmChart   HelmChartReference `json:"helmChart,omitempty"`
```

Comment:
`HelmChartReference` and `TemplateReference` should be off the pointer type, as only one or another can be present.